android/api-tests/src/test/java/com/revenuecat/apitests/kotlin/CommonApiTests.kt

```diff
@@ -173,8 +173,9 @@ private class CommonApiTests {
         getCustomerInfo(onResult)
     }
 
-    fun checkSyncPurchases() {
+    fun checkSyncPurchases(onResult: OnResult) {
         syncPurchases()
+        syncPurchases(onResult)
```

Comment:
Shouldn't this replace the existing syncPurchases() call so we don't call syncPurchases twice?

Reply:
Kotlin method overload means both methods should be fine to live side by side

Reply:
fwiw you could do:

```
@JvmOverloads
fun syncPurchases(
    onResult: OnResult? = null,
) {
```

to only have a single `syncPurchases`, but that requires adding logic inside to check which `syncPurchases` function to call, so I think what you  did is also ok

Reply:
@MarkVillacampa This was my bad, I thought that this change was added somewhere in the codebase's `syncPurchases()` function, but I now realize that this is in an API tester so it's totally okay :D